KNOWN_ISSUES.md

```diff
@@ -10,23 +10,26 @@ the registry to see if there might be another.
 
 
 
-##  PyVO regsearch() keywords argument usage
-
-
-Note also that each string in the list given to PyVO's regsearch() keywords argument is searched in the subject, description, and title of the resource. 
-
-**Workaround**:   If you want to search for the ivoid/identity, you have to do this after the fact as described below.  
+##  PyVO regsearch() update:  
 
+PyVO's registry search functions have been updated to allow for more
+powerful methods of data discovery.  Previously you could
+use arguments keywords, servicetype, and waveband, and these still work:
 ```
-image_services = vo.regsearch(servicetype='image', keywords=['sloan'])
-
-sdss_gavo_service = [s for s in image_services if 'gavo' in s.ivoid][0]
-
-sdss_gavo_service.search(query)
+image_services = vo.regsearch(servicetype='image', keywords=['sloan vla'])
 ```
+This searches for the strings 'sloan' AND 'vla' in the same entry.  If you instead used `keywords=['sloan','vla']`,
+it would return records with either string, not only both.  But you
+can also specify more accurate search criteria using new PyVO Registry
+methods.  You can now also search for particular information in a
+catalog by its UCD, from a particular source by it's IVOID, and by
+spatial, spectral, or temporal coverage.  Note however that where such
+coverage information is not provided in the Registry (which it often
+is not), results will be returned anyway.  
```

Comment:
I am not sure I understand this: `results will be returned anyway`. Can you clarify please.

Also, isn't there an issue with other search criterion too. If the registry doesn't have information about the UCDs available in the catalog, then it will not show up.

Reply:
Well, I was pretty sure we're not specifying the regions covered by each catalog.  And if you search for services with coverage of a given location, you sometimes get them back whether they have data in that region or not.  In the case of HEASARC services, we apparently specify "all sky" when the reality is "maybe a variety of places all over the sky" or just "we didn't specify".  I'm trying to figure out if this is an issue broader than HEASARC.  But usage of this coverage info is maybe something we should talk about and test more before we advertise it.

Reply:
I rephrased in a way that mentions the issue without specifically calling out mistakes HEASARC may or may not be making.  ;)

Reply:
> @trjaffe,
> This is somewhat similar to the tests I reported in slack for UCDs. It looks like pyvo is one step ahead of the services provided.

Even if the service is providing the spatial info to the registry, the user has to be careful.  E.g., on another service, GAVO has the spatial info given down to HiPS resolution level 3 (which if I'm doing the math right isn't great, about 5 degrees wide).  And it returns no results in the area the registry says it should.  This isn't surprising I guess, but we should work with it some more so we can tell users how to use it.